src/main/java/org/n52/wacodis/javaps/preprocessing/ReferenceDataPreprocessor.java

```diff
@@ -106,7 +108,13 @@ public void setEpsg(String epsg) {
      * @throws WacodisProcessingException
      */
     @Override
-    public List<File> preprocess(SimpleFeatureCollection inputCollection, String outputDirectoryPath) throws WacodisProcessingException {
+    public List<File> preprocess(SimpleFeatureCollection inputCollection, String outputDirectoryPath, String targetepsg) throws WacodisProcessingException {
```

Comment:
Provide EPSG code when instantiating the _ReferenceDataPreprocessor_ to avoid changing the _InputDataPreprocessor_ interface.

Reply:
Changed the implementation to reproject the images
Now the EPSG-Code is a parameter in the HashMap